src/client/src/models/Employee.js

```diff
@@ -1,12 +1,16 @@
 export default class Employee {
-    
-    constructor(id, name) {
-        this.id=id;
-        this.name=name;
-    }
 
-    edit(id, name) {
-        this.id = id;
-        this.name = name;
-    }
+        constructor(id, first_name, last_name, email) {
+            this.id=id;
+            this.name=name;
+            this.firstName=first_name;
+            this.lastName=last_name;
+            this.email=email;
+        }
+
+        edit(first_name, last_name, email) {
```

Comment:
Please rename this function to modifyEmployee(first_name, last_name, email)

Reason: we  need consistency in the function names (other classes have the function modifyObjectName() )